### DISCUSSION TITLE
GHA based CI use tips

### DISCUSSION THREAD

**Discussion by LABPBS** [msys2/MINGW-packages#18239] [Q&A]
I've forked this repo to use the GHA based CI for custom builds more times than i can count and i sincerely thank its developers, i have a question or 2

suppose i wanted to do weekly builds of qbittorrent with libtorrent branch 1.2, ive managed to get this working by commiting to my fork, but how would i do that on a weekly basis? this isnt such a deal for a project like qbittorrent that doesnt have that much commit frequency on its stable/released branches but it would be quite interesting for a project like transmission that currently has a high commit frequency

thanks folks 😄

**Comment by Biswa96** [comment]
> how would i do that on a weekly basis? 

1. Create your own repository.
2. Add the required files to build the project.
3. Set the CI job to build weekly.
